Cargo.toml

```diff
@@ -1,13 +1,21 @@
-[workspace]
-resolver = "2"
-members = [
-    "derive",
-    "lib",
-    "sr25519",
-]
-
-[workspace.package]
-version = "0.3.7"
+[package]
+name = "gear-lib"
+version = "0.4.1"
 edition = "2021"
 license = "MIT"
 authors = ["Gear Technologies"]
+
+[dependencies]
+gstd = { git = "https://github.com/gear-tech/gear", rev = "78dfa07" }
```

Comment:
Since GitHub redirects repo links without `.git` to ones with it, it's safe to do so. And looks shorter & better.

Reply:
Agree about redirection, but (in theory) there is no only GitHub for git 😄 
Also, these links are copied to `Cargo.lock` without modifications, and that yields a mixed style in `Cargo.lock`. Of course, it is more convention rather than a requirement, but I'd follow the official style. But not critical indeed.